### PR TITLE
change response headers to 'application/json' for json

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: src/controllers/InstallerAPI.ts [hunk 1]
```diff
@@ -252,6 +256,7 @@ export class Installers {
   }
 
   async doMakeInstaller(response: Express.Response, request: Express.Request, id: string, slug: string, skipValidation: boolean): Promise<string | ErrorResponse> {
+    response.contentType("application/json");
```
Comment (emosbaugh) [inline src/controllers/InstallerAPI.ts:259]: This function returns plain text response and sets `response.contentType("text/plain");` on line 318. Does that properly override the content type?

File: src/controllers/InstallerAPI.ts [hunk 2]
```diff
@@ -227,6 +230,7 @@ export class Installers {
     @Res() response: Express.Response,
     @Req() request: Express.Request,
   ): Promise<string | ErrorResponse> {
+    response.contentType("application/json");
```
Comment (emosbaugh) [inline src/controllers/InstallerAPI.ts:233]: should you set the content type to text/plain on line 255 before you return the empty string?

Comment (laverya) [reply src/controllers/InstallerAPI.ts:233]: Is an empty string not json?

Comment (laverya) [reply src/controllers/InstallerAPI.ts:233]: (the consumer doesn't care here, though)

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**